app/src/main/res/values-te-rIN/strings.xml

```diff
@@ -0,0 +1,264 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="action_about_us">మా గురించి చర్య</string>
+    <string name="dialog_cancel_sign_up">డైలాగ్ సైన్అప్ రద్దు</string>
+    <string name="menu_item_about">మెను ఐటెమ్ గురించి</string>
+    <string name="menu_item_settings">మెను ఐటమ్ సెట్టింగులు</string>
```

Comment:
This is like the word to word translation.
I think it has to be just "settings" instead of "menu item settings" in the translation
I think most of these are in the same way.

Reply:
okay....will correct it